docs/technical_reference/costing/electrolyzer.rst

```diff
@@ -19,35 +24,33 @@ The following variables are constructed on the unit block (e.g., m.fs.unit.costi
 .. csv-table::
    :header: "Description", "Symbol", "Variable Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "variable_name", "1", ":math:`\text{dimensionless}`"
+   "Total unit capital cost", ":math:`C_{cap}`", "capital_cost", "10,000", ":math:`\text{USD}_{2020}`"
+   "Fixed operating costs", ":math:`C_{op}`", "fixed_operating_cost", "10,000", ":math:`\text{USD}_{2020}\text{/yr}`"
+   "Membrane capital cost", ":math:`C_{cap,\, mem}`", "membrane_cost", "10,000", ":math:`\text{USD}_{2020}`"
+   "Anode capital cost", ":math:`C_{cap,\, anode}`", "anode_cost", "10,000", ":math:`\text{USD}_{2020}`"
+   "Cathode capital cost", ":math:`C_{cap,\, cathode}`", "cathode_cost", "10,000", ":math:`\text{USD}_{2020}`"
+   "Membrane replacement cost", ":math:`C_{op,\, mem}`", "membrane_replacement_cost", "10,000", ":math:`\text{USD}_{2020}\text{/yr}`"
```

Comment:
Forgot to mention this in my review, but we were planning on getting rid of the default value column in this table and replacing it with "Index" since a lot of these variables didn't have default values (as indicated by your placeholders) Adam will put up a PR that updates this for the auto-generated files, but it won't touch documentation that has already been updated. So we should update this before merging.

Reply:
@MarcusHolly I agree that default doesn't make sense here because it is basically the values that are initialized. However, I also don't know if any of these variables are indexed across any of the unit models, although in theory they could be. I went ahead and edited the Electrolyzer and GAC in this to lessen the number of PRs.